lib/tasks/db_maintenance.rake

```diff
@@ -236,7 +236,12 @@ namespace :cartodb do
       execute_on_users_with_index(task_name, Proc.new { |user, i|
         begin
           log(sprintf("Trying on %-#{20}s %-#{20}s (%-#{4}s/%-#{4}s)...", user.username, user.database_name, i+1, count), task_name, database_host)
-          user.db_service.load_cartodb_functions(statement_timeout, extension_version)
+          # Only load cartodb extensions once per database, i.e. for dedicated users and org owners
+          load_cartodb_extension = user.account_type == '[DEDICATED]' || user.organization_owner?
```

Comment:
Org owner is already dedicated type.. Following should be enough --->
load_cartodb_extension = user.account_type == '[DEDICATED]'

Reply:
This is true, but I chose this because it's a bit more readable/failsafe.